fastlane/lib/fastlane/plugins/plugin_generator_ui.rb

```diff
@@ -0,0 +1,15 @@
+module Fastlane
+  class PluginGeneratorUI
```

Comment:
Why do we need a new UI class instead of using the one from `fastlane_core`? We can also easily mock the `UI` class from core.